build-tools/src/main/resources/software/amazon/awssdk/to-string-velocity-template.vm

```diff
@@ -0,0 +1,11 @@
+public java.lang.String toString() {
```

Comment:
is this actually used anywhere? i can't see it referenced...

Reply:
It's an intellij velocity template to configure it to auto-generate `toString` implementations using `ToStringResult`.

I can add intellij to the name to make it clearer, even though it will probably work for other IDEs.

Reply:
ah - sorry had missed it's intent. can we comment it?

Reply:
Does it work in eclipse? Otherwise, include intellij in the name?

Reply:
Yes! http://velocity.apache.org/engine/1.7/vtl-reference.html#comments
I'll add a comment on how to use it.